frame/staking-rewards/src/test/runtime.rs

```diff
@@ -172,7 +172,7 @@ parameter_types! {
 	pub const NativeAssetId: CurrencyId = PICA::ID;
 }
 
-impl pallet_assets::Config for Runtime {
+impl pallet_assets::Config for Test {
```

Comment:
i am agains it . it is NOT Test. It is Runtime for Test. Why you name Runtime to be Test? It is not test tests. Right? So it uses simplfied Runtime.

Reply:
I integrate this pallet into benches and into Dali, it is much easier if Runtime is named same in all places. please change back.

Reply:
I'm just following the convention among substrate developers, which makes sense to me to differentiate between a Test runtime and an actual runtime when searching through code.

Reply:
please send me reference

Reply:
Just check the frame pallets in substrate repo